pysipp/agent.py

```diff
@@ -39,7 +39,7 @@ class UserAgent(command.SippCmd):
     configuration options for a SIP UA.
     """
     # we skip `error` since we can get it from stderr
-    _log_types = 'screen calldebug message log'.split()
+    _log_types = 'screen calldebug message log error'.split()
```

Comment:
I have a better idea on the cause of the error.

The uas SIPp process has no address parameter and SIPp fails very quickly.

pysipp detects the failure and kills the uac SIPp process before SIPp 
creates the error file.

Adding "time.sleep(1.5)" at line 78 of launch.py, just after the loop 
that starts the SIPp processes, makes all tests pass.

But I think it is not a good idea to delay the waiter thread starting. 
All tests pass, but I am not sure it does induce other issues.

Le 26/07/2016 à 16:55, goodboy a écrit :
>
> In pysipp/agent.py 
> <https://github.com/SIPp/pysipp/pull/19#discussion_r72265805>:
>
> > @@ -39,7 +39,7 @@ class UserAgent(command.SippCmd):
> >      configuration options for a SIP UA.
> >      """
> >      # we skip `error` since we can get it from stderr
> > -    _log_types = 'screen calldebug message log'.split()
> > +    _log_types = 'screen calldebug message log error'.split()
>
> How were you able to have the test which is failing now in the 
> |TravisCI| to work?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/SIPp/pysipp/pull/19/files/271d6193be712c3342e3f137cd4c678aee6cf75c#r72265805>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AToJTYyCQmt-1fPo-zGiqwP2wXTw1wG4ks5qZh_GgaJpZM4JSigv>.
>